packages/core/config/babel-preset.js

```diff
@@ -148,7 +162,29 @@ module.exports = () => {
               ],
             },
           ],
-          ['inline-react-svg'],
+          [
+            'inline-react-svg',
+            {
+              svgo: {
+                plugins: extendDefaultPlugins([
+                  {
+                    name: 'removeAttrs',
+                    params: { attrs: '(data-name)' },
+                  },
+                  {
+                    // @TODO confirm this is the right thing
```

Comment:
How do we confirm that this is the right thing?

Reply:
I checked some other svgs from heroicons/feathericons - they seem to include viewBox. For now I'd suggest lets keep it in (because its backwards compatible)